site-src/v1alpha2/references/policy-attachment.md

```diff
@@ -0,0 +1,290 @@
+# Policy Attachment
+
+Many implementations of the API support additional functionality that is not
+covered by the API directly. For example, concepts like timeouts, retries, or
+even CDN configuration are not sufficiently portable to be defined within the
```

Comment:
nit: load-balancing, health-checks might be more suited examples than CDN.

Reply:
Updated, thanks!